konan/konan.properties

```diff
@@ -510,7 +510,8 @@ abiSpecificLibraries.linux_arm32_hfp = lib usr/lib
 # targetSysRoot relative
 crtFilesLocation.linux_arm32_hfp = usr/lib
 runtimeDefinitions.linux_arm32_hfp = USE_GCC_UNWIND=1 KONAN_LINUX=1 \
-  KONAN_ARM32=1 USE_ELF_SYMBOLS=1 ELFSIZE=32 KONAN_NO_UNALIGNED_ACCESS=1
+  KONAN_ARM32=1 USE_ELF_SYMBOLS=1 ELFSIZE=32 KONAN_NO_UNALIGNED_ACCESS=1 \
+  KONAN_HAS_CXX11_EXCEPTION_FUNCTIONS=1
```

Comment:
Maybe it's more prudent to use a reverse property

Reply:
Or just get rid of this obsolete property at all.